integration-tests/docker-compose.yml

```diff
@@ -0,0 +1,99 @@
+version: '3.7'
```

Comment:
Why are you adding this file here if `docker-compose.yml` is already in the `docker` directory?

Reply:
Done.